common/dynamicconfig/constants.go

```diff
@@ -3871,6 +3879,11 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "Enable shardId metrics in persistence client",
 		DefaultValue: true,
 	},
+	EnableTTL: DynamicBool{
+		KeyName:      "enableTTL",
```

Comment:
the key name should follow the existing naming pattern.
We can name it as `system.enableExecutionTTL`